package.json

```diff
@@ -52,6 +50,8 @@
     "lodash": "^4.17.4",
     "phantomjs-prebuilt": "^2.1.15",
     "semver": "^5.3.0",
+    "standard-version": "^7.0.1",
+    "validate-commits": "git+ssh://git@github.com/bigcommerce-labs/validate-commits.git#1.3.0",
```

Comment:
I don't think you need this here since we only validate the commits in CircleCI, and we pull the package directly there.

Reply:
fixed